packages/fontless/src/vite.ts

```diff
@@ -64,20 +70,28 @@ export function fontless(_options?: FontlessOptions): Plugin {
         cssTransformOptions.esbuildOptions = defu(cssTransformOptions.esbuildOptions, resolveMinifyCssEsbuildOptions(config.esbuild))
       }
     },
-    async transform(code, id) {
-      // Early return if no font-family is used in this CSS
-      if (!options.processCSSVariables && !code.includes('font-family:')) {
-        return
-      }
+    transform: {
+      filter: {
+        id: {
+          include: [CSS_EXTENSIONS_RE, CSS_LANG_QUERY_RE, INLINE_STYLE_ID_RE],
+          exclude: [/.vite/],
```

Comment:
wouldn't this exclude diretories with `vite` in the name?

Reply:
Nice catch @danielroe :rocket: